nautobot/dcim/tables/devices.py

```diff
@@ -1421,9 +1423,14 @@ class Meta(BaseTable.Meta):
             "platform",
             "role",
             "tenant",
+            "capabilities",
             "actions",
         )
 
+    def render_capabilities(self, value):
+        """Render capabilities."""
+        return ", ".join(value)
```

Comment:
Since these are fixed choices, it *might* be nice to render them similar to how we render multiselect custom fields (CustomFieldColumn in nautobot/core/tables.py), i.e. as `<span class="label label-default">{value}</span>`.